plugins/command-manager/src/main/java/com/wazuh/commandmanager/settings/PluginSettings.java

```diff
@@ -81,10 +85,21 @@ private PluginSettings(@NonNull final Settings settings) {
         this.timeout = CLIENT_TIMEOUT.get(settings);
         this.jobSchedule = JOB_SCHEDULE.get(settings);
         this.jobPageSize = JOB_PAGE_SIZE.get(settings);
-        this.jobKeepAlive = JOB_KEEP_ALIVE.get(settings);
+        this.pitKeepAlive = this.jobSchedule * 60;
+        this.validateSettings();
         log.debug("Settings loaded: {}", this.toString());
     }
 
+    /** Fits setting values to the internal logic */
+    private void validateSettings() {
+        // If timeout is not less than job period in seconds
+        if (!(this.timeout < this.jobSchedule * 60)) {
+            // Set timeout to half job period (in seconds)
+            this.timeout = this.jobSchedule * 30;
+            log.info("Setting command_manager.client.timeout to half command_manager.job.schedule (in seconds): {}", this.timeout);
```

Comment:
I'd be confused seeing this log, as no reason is given. We could maybe add 2 logs

- warn: notifies about invalid/unrecommended settings
- info: notifies about the action taken to fix that